sink_service/source/main.c

```diff
@@ -60,17 +60,79 @@ static bool get_service_name(char service_name[MAX_SIZE_SERVICE_NAME], unsigned
     return true;
 }
 
+/**
+ * \brief   Attempts to acquire an environment variable
+ * \param   dest
+ *          The destination buffer where to store the read value
+ * \param   name
+ *          The name of the environment variable to acquire
+ * \return  A pointer to the character buffer
+ */
+static char * get_env_var(char * name)
+{
+    char * ptr = NULL;
+    if ((ptr = getenv(name)) != NULL)
+    {
+        LOGI("%s: %lu\n", name, ptr);
+    }
+    return ptr
+}
+
+/**
+ * \brief   Obtains environment parameters to control sink settings
+ * \param   bitrate
+ *          Pointer where to store bitrate value (if any)
+ * \param   port_name
+ *          Pointer where to store port_name value (if any)
+ * \param   sink_id
+ *          Pointer where to store sink_id value (if any)
+ * \param   max_poll_fail_duration
+ *          Pointer where to store max_poll_fail_duration value (if any)
+ */
+static void get_env_parameters(unsigned long * bitrate,
+                               char * port_name,
```

Comment:
It should be char ** port_name_p, right? (I just added _p suffix to distinguish that it is a pointer to the var